File: Ass_16/Assignment_1.c
#include<stdio.h>

void Pattern(int iRow, int iCol)
{
    int i = 0;
    int j = 0;

    for(int i = 1; i <= iRow; i++)
    {
        for(j =1; j <= iCol; j++)
        {
            if(i >= j)
            {
                printf("*\t");
            }
            
        }
        printf("\n");
    }
    printf("\n");
}
int main()
{
    int iValue1 = 0;
    int iValue2 = 0;

    printf("Enter number of rows :");
    scanf("%d",&iValue1);

    printf("Enter number of column:");
    scanf("%d",&iValue2);

    Pattern(iValue1,iValue2);

    return 0;


}

File: Ass_47/Assignment_1.java
import java.util.*;

class Pattern
{
    public void Pattern(int iRow,int iCol)
    {
        int i = 0, j = 0;

        for(i = 0;i <iRow;i++)
        {
            for(j = 0; j <iCol;j++)
            {
                System.out.print(" * ");
               
            }
            System.out.println("\n");

        }
        
    }
}
class Assignment_1
{
    public static void main(String[]args)
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter the number of rows :");
        int iRow = sobj.nextInt();

        System.out.println("Enter the number of columns :");
        int iCol = sobj.nextInt(); 

        Pattern pobj = new Pattern();
        pobj.Pattern(iRow,iCol);
    }
}